commonmark/src/test/java/org/commonmark/test/LinkReferenceDefinitionTest.java

```diff
@@ -0,0 +1,64 @@
+package org.commonmark.test;
+
+import org.commonmark.node.Node;
+import org.commonmark.node.Link;
+import org.commonmark.node.AutoLink;
+import org.commonmark.node.Visitor;
+import org.commonmark.node.AbstractVisitor;
+import org.commonmark.parser.Parser;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.commonmark.internal.util.Debugging.log;
+import static org.commonmark.internal.util.Debugging.toStringTree;
+
+public class LinkReferenceDefinitionTest {
+
+    @Test
+    public void one() {
+
+        final Parser.Builder builder = Parser.builder();
+        final Parser parser = builder.build();
+        final Node document = parser.parse(getText());
+        final java.util.List<Link> links = new java.util.ArrayList();
```

Comment:
Add imports for `java.util.List` and `java.util.ArrayList`. Also, please add generics (`new ArrayList<>();`).